### PR TITLE
Add import wallet screens

### PR DESCRIPTION
**Pull request by DHRUVA-N-L-CHOUDHARY**

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -66,7 +66,7 @@ android {
 
     splits {
         abi {
-          enable true
```
Comment (kamikazechaser) [inline android/app/build.gradle]: This will break the builds. Revert back to true

**Review by BlazingFV — COMMENTED** [review]
could you please share some screen shots for the final version of wallet ? so i can cross check .
thanks

**Comment by DHRUVA-N-L-CHOUDHARY** [comment]
image
image

**Comment by BlazingFV** [comment]
okay looks good enough I'll approve it
the code alsk seems fine

**Review by BlazingFV — APPROVED** [review]
good work everything looks fine and the ui looks like
the figma design

File: android/build.gradle [hunk 2]
```diff
@@ -22,7 +22,8 @@ rootProject.buildDir = '../build'
 subprojects {
     project.buildDir = "${rootProject.buildDir}/${project.name}"
 }
-subprojects {
+subprojects
```
Comment (kamikazechaser) [inline android/build.gradle]: Prefer bracket on the same line

File: .github/workflows/analyze-build-debug.yaml [hunk 3]
```diff
@@ -13,6 +13,10 @@ jobs:
     name: Flutter analyze, build
     runs-on: ubuntu-latest
     steps:
+      - name: Save state
```
Comment (kamikazechaser) [inline .github/workflows/analyze-build-debug.yaml]: Remove the next 2 steps. echoing random values on CI is not a good practice especially from a security PoV.

File: lib/base_widgets/custom_input_box.dart [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+import 'package:grassroots_flutter/constants/custom_color.dart';
+
+class CustomTextfield extends StatefulWidget {
+  const CustomTextfield({super.key});
+
+  @override
+  State<CustomTextfield> createState() => _CustomTextfieldState();
+}
+
+class _CustomTextfieldState extends State<CustomTextfield> {
```
Comment (kamikazechaser) [inline lib/base_widgets/custom_input_box.dart:14]: @sanjh Kindly review this base widget.

**Review by kamikazechaser — CHANGES_REQUESTED** [review]

**Comment by kamikazechaser** [comment]
- Remove empty files
- Move import wallet screen to onboarding folder or its own wallet folder

**Pull request merged**